### DISCUSSION TITLE
优雅的 RACCommand · /raccommand

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#264] [Announcements]
https://draveness.me/raccommand

**Comment by poholo** [comment]
对RACCommand还是不理解，看来得多补补其他资料，回头再读这篇，图文并茂。

**Comment by liudeping** [comment]
immediateExecuting 中 map的作用应该将大于0 作为YES

**Comment by yinfan198741** [comment]
文章写的不够清晰,说句实话啊,感觉就是大部分翻译,没有抓住重点,没从上到下的剖析.

**Comment by draveness** [comment]
> @yinfan198741
> 文章写的不够清晰,说句实话啊,感觉就是大部分翻译,没有抓住重点,没从上到下的剖析.

你说的有道理

**Comment by yinfan198741** [comment]
> ,没从上到下的剖析.

我觉得
1#首先要画一个整体的布局图,这个布局图是一个大概的,只是说有大概有那些组件.
2#要说明这个东西为什么存在,为什么command包装了一个RACSignal,我们用它有说明好处.
3#渐进式的编写一些用例,且都能展现Command的特性的用例.
4#剖析每个组件里面的原理
这个是我觉得比较好的方式,如果写的不对,忽略

**Comment by draveness** [comment]
> > ,没从上到下的剖析.
> 
> 我觉得
> 1#首先要画一个整体的布局图,这个布局图是一个大概的,只是说有大概有那些组件.
> 2#要说明这个东西为什么存在,为什么command包装了一个RACSignal,我们用它有说明好处.
> 3#渐进式的编写一些用例,且都能展现Command的特性的用例.
> 4#剖析每个组件里面的原理
> 这个是我觉得比较好的方式,如果写的不对,忽略

看下最新文章吧，不写 iOS 了
